index.bs

```diff
@@ -263,6 +263,30 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing browsers.
+
+Content relying on existing browser behavior can take many forms.
+It may rely on elements, attributes or APIs that are part of earlier specifications, or on features that are entirely proprietary.
+It may depend on specific error handling rules. It may depend on certain method names not being defined.
+In rare cases, it may even depend on a feature from earlier specifications not being implemented as specified.
+
+When considering changes to legacy features or behavior, relative to current implementations and author expectations, the following questions should be considered:
+
+* Does a significant quantity of existing content depend on the feature or behavior?
+* Does any of the dependent content occur on particularly popular websites?
```

Comment:
I am not a big fan or talking about popularity. An intranet site at a large might be just as important. It might also be used on a very unpopular website that everyone uses, like for declaring taxes ;-)

maybe "high volume websites or intranet sites"

We have made such changes before in such cases, what really mattered was making a public plan and giving people time to adjust existing content

Reply:
```suggestion
* Does any of the dependent content occur on particularly high volume websites?
```